content/source/layouts/enterprise2.erb

```diff
@@ -156,6 +156,10 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("docs-enterprise2-logging") %>>
```

Comment:
I think the nav link ought to go inside the "private Terraform Enterprise" section, since it's not applicable to saas.

Reply:
The only reason I put it here is because the SAML was also there? I can move it though.